deploy_website.sh

```diff
@@ -25,10 +25,10 @@ cd $DIR
 
 # Fix *.md links to point to where the docs live under Mkdocs.
 # Linux
-# sed -i 's/docs\/wire_compiler.md/wire_compiler/' README.md
+# sed -i 's/docs\/wire_compiler.md/wire_compiler.md/' README.md
 # OSX
-sed -i "" 's/docs\/wire_compiler.md/wire_compiler/' README.md
-sed -i "" 's/docs\/wire_grpc.md/wire_grpc/' README.md
```

Comment:
It works now, but  `mkdocs`  emits a warning:
```
Doc file 'index.md' contains an unrecognized relative link 'wire_compiler', it was left as is.
```
Because it expects all internal links to be links to existing files.
This change is only to fix this warning.